packages/data-context/src/data/coreDataShape.ts

```diff
@@ -199,7 +199,7 @@ export function makeCoreData (modeOptions: Partial<AllModeOptions> = {}): CoreDa
       detectedBundler: null,
       detectedFramework: null,
       // TODO: API to add third party frameworks to this list.
-      frameworks: CT_FRAMEWORKS.map((framework) => resolveComponentFrameworkDefinition(framework)),
+      frameworks: [],
```

Comment:
Is `wizard.initialize` guaranteed to have been called? TODO...

Reply:
I'll look around for a better place to push these frameworks in. I do think we need the official ones populated initially for cases where they already have it configured (deps warnings, create from component)